### PR TITLE
Update Coinspect blog link to https .com domain

### PR DESCRIPTION
**Pull request by juli**

### CODE REVIEW

File: _posts/2014-10-07-socratic-seminar-30.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Weekly Mining Stats
 [](http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html)[http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html](http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html)
 
 Building a block-chain database the right way  
-[](http://blog.coinspect.co/building-block-chain-database-right-way)[http://blog.coinspect.co/building-block-chain-database-right-way](http://blog.coinspect.co/building-block-chain-database-right-way)
+[](https://www.coinspect.com/blog/building-block-chain-database-right-way/)[[https://www.coinspect.com/blog/building-block-chain-database-right-way/](https://www.coinspect.com/blog/building-block-chain-database-right-way/)](https://www.coinspect.com/blog/building-block-chain-database-right-way/)
```
Comment (murchandamus) [inline _posts/2014-10-07-socratic-seminar-30.md:56]: I’m not sure why the links are all in here with an empty link text and repeated in full, but this introduces a new unnecessary opening bracket that should be dropped.

```suggestion
[](https://www.coinspect.com/blog/building-block-chain-database-right-way/)[https://www.coinspect.com/blog/building-block-chain-database-right-way/](https://www.coinspect.com/blog/building-block-chain-database-right-way/)](https://www.coinspect.com/blog/building-block-chain-database-right-way/)
```

**Review by murchandamus — COMMENTED** [review]
ACK

I have verified that the original link now forwards to the one that is being proposed here.

**Comment by murchandamus** [comment]
Hey @juli, what are you trying to do, anything I can help with?

**Comment by juli** [comment]
Hey sorry, I accepted the commit removing the bracket but it was wrong so I reverted and started again

**Comment by murchandamus** [comment]
I see, I’ll give you some time to take a stab at it if you want to, but let me know if you have any questions or if you’d like me to take over.

**Comment by juli** [comment]
feel free to take over! the change was simple but the empty `[]` was confusing, sorry. We just wanted to avoid the redirect both links work.

**Comment by murchandamus** [comment]
Ah, I see, I was confused. I was looking at the line with the force push which made me think that you had reverted back to the original, but the commit after the force push had already fixed it all.

**Pull request merged**